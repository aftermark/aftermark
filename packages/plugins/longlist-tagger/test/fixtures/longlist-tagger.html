<!DOCTYPE html>
<body>
  <ul>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ul>
  <ol>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ol>
  <ul>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ul>
  <ol>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ol>
  <ul>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ul>
  <ol>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ol>
</body>
