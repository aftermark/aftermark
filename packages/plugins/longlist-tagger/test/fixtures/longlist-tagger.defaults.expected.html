<!DOCTYPE html>
<body>
  <ul>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ul>
  <ol>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ol>
  <ul>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ul>
  <ol>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ol>
  <ul class="long-list">
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ul>
  <ol class="long-list">
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
    <li>Item</li>
  </ol>
</body>
